doc/source/configuration/wazuh.rst

```diff
@@ -0,0 +1,330 @@
+If using environments it ends up in ``$KAYOBE_CONFIG_PATH/environments/<env_name>/wazuh-secrets.yml``
+Remember to encrypt!
+
+Wazuh secrets template is located in ``etc/kayobe/ansible/templates/wazuh-secrets.yml.j2``.
+It will be used by wazuh secrets playbook to generate wazuh secrets vault file.
+
+
+.. code-block:: console
+
+  kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/wazuh-secrets.yml
+  ansible-vault encrypt --vault-password-file ~/vault.pass $KAYOBE_CONFIG_PATH/inventory/group_vars/wazuh/wazuh-manager/wazuh-secrets
+
+
+TLS (optional)
+--------------
+
+You can generate your own TLS certificates, otherwise skip this section.
+By default, Wazuh Ansible uses `wazuh-cert-tool.sh <https://documentation.wazuh.com/current/user-manual/certificates.html>`__
+to automatically
+generate certificates for wazuh-indexer (previously Elasticsearch and opendistro)
+and wazuh-dashbooard (previously Kibana) using a local CA.
+If the certificates directory ``etc/kayobe/ansible/wazuh/certificates``
+does not exist, it will generate the following certificates in ``etc/kayobe/ansible/wazuh/certificates/certs/``
+(here os-wazuh is set as ``elasticsearch_node_name`` and ``kibana_node_name``:
+
+
+* Admin certificate for opendistro security
+   * admin-key.pem,  admin.pem
+* Node certificate
+   * os-wazuh-key.pem,  os-wazuh.pem
+* HTTP certificate for Kibana (port 5601) & Elasticsearch (port 9200)
+   * os-wazuh_http.key, os-wazuh_http.pem
+* Root CA certificate
+   * root-ca.key  root-ca.pem
+
+
+
+It is also possible to use externally generated certificates for wazuh-dashboard. root-ca.pem should contain the CA chain.
+Those certificates can be uploaded to ``etc/kayobe/ansible/wazuh/custom_certificates``,
+and will replace certificates generated by wazuh.
+Certificates should have the same name scheme as those generated by wazuh (typicaly <node-name>.pem)
+The key for the external certificate should be in PKCS#8 format
+(in its header it may have BEGIN PRIVATE KEY instead of BEGIN RSA PRIVATE KEY or BEGIN OPENSSH PRIVATE KEY).
+
+Example OpenSSL rune to convert to PKCS#8:
+
+``openssl pkcs8 -topk8 -nocrypt -in wazuh.key -out wazuh.key.pkcs8``
+
+TODO: document how to use a local certificate. Do we need to override all certificates?
+
+Deploy
+------
+
+Deploy Wazuh manager:
+
+``kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/wazuh-manager.yml``
+
+If you are using the wazuh generated certificates,
+this will result in the creation of some certificates and keys (in case of custom certs adjust path to it).
+Encrypt the keys (and remember to commit to git):
+
+
+``ansible-vault encrypt --vault-password-file ~/vault.pass $KAYOBE_CONFIG_PATH/ansible/wazuh/certificates/certs/*.key``
+
+Verification
+==============
+
+The Wazuh portal should be accessible on port 443 of the Wazuh
+manager’s IPs (using HTTPS, with the root CA cert in ``etc/kayobe/ansible/wazuh/certificates/root-ca.pem``).
```

Comment:
Is it in etc/kayobe/ansible/wazuh/certificates/certs/root-ca.pem?

Reply:
Fixed